### PR TITLE
Add PATCH support

### PR DESCRIPTION
**Pull request by hernangonzalez**
Hi,

Added support to PATCH methods that I use in my projects. Let me know if anything needs a change.

Thanks,
H.

### CODE REVIEW

File: Source/Router.swift
Comment (joanromano) [inline Source/Router.swift]: Please indent this properly

File: Tests/RouterTests.swift
Comment (joanromano) [inline Tests/RouterTests.swift]: Also in this file indentations are wrong, could you please fix them?

**Review by MP0w — COMMENTED** [review]
I accidentally discarded my previous comment:
Looks like indentation is broken, maybe you have different settings. Please take look.
Other than this LGTM, ty for the PR 👍

**Comment by joanromano** [comment]
Also fine for me as soon as you fix the indentation issues. Special kudos for adding the unit test and keep the coverage up to date 👍

**Comment by hernangonzalez** [comment]
@MP0w @joanromano updated 👍 , thanks.

**Review by MP0w — APPROVED** [review]

File: Source/Router.swift
Comment (MP0w) [inline Source/Router.swift]: oh apparently all of this is still shifted

File: Source/Router.swift
Comment (MP0w) [inline Source/Router.swift]: and this line still has a tab I guess

File: Source/Router.swift
Comment (MP0w) [reply Source/Router.swift]: https://github.com/WhimDatingApp/Kakapo/blob/0684511fc4197081e50fe3f19560817907a8f2a0/Source/Router.swift#L288

Comment (hernangonzalez) [reply Source/Router.swift]: ups 😢

**Comment by hernangonzalez** [comment]
Sublime to the rescue 🙏

**Review by joanromano — APPROVED** [review]
Looks like we are good to go! Thanks for the contribution 🍺 🍺 🍺

**Review by MP0w — APPROVED** [review]
ty, let's just wait for ci and I will merge 🎉 
https://www.youtube.com/watch?v=SsoOG6ZeyUI :)

File: Source/Router.swift
Comment (MP0w) [reply Source/Router.swift]: @joanromano I think we misuse the review... you add comments but people won't see it until you "finish" the review with one of the options (comment, approve, decline)...
so we couldn't see your comments until you approved 👎

Comment (MP0w) [reply Source/Router.swift]: FYI I think I did the same mistake before

Comment (joanromano) [reply Source/Router.swift]: oh, my bad then 😞 Sorry if that confused you @hernangonzalez

**Comment by MP0w** [comment]
Travis has issues with macOS today, builds are delayed

**Pull request merged**
